lib/matplotlib/figure.py

```diff
@@ -1163,6 +1164,31 @@ def colorbar(
         self.stale = True
         return cb
 
+    @docstring.dedent_interpd
+    def bivariate_legend(self, *args, ax=None, legend_padding=None,
+                         subplot_adjust=None, **kwargs):
+        if (subplot_adjust is None):
```

Comment:
No matter where this lands, I think that the legend itself should be passed an axes argument - its really to big to be a decoration on an existing axes.  Even if the axes is created with `inset_axes`.    `subplots_adjust` is incompatible with constrained layout, and should rarely be necessary.